plugins/helper.js

```diff
@@ -0,0 +1,24 @@
+import { inject } from "vue"
+
+import moment from "moment"
+
+const helper = {
+    getCurrentTimeInMS: () => {
+        return Date.now()
+    },
+
+    objectDeepClone: (obj = {}) => {
+        if (typeof obj === "object") {
```

Comment:
if (typeof obj === "object") 
  
  what if we pass an array to this function

Reply:
typeof array is also object. To specifically check for array we can use Array.isArray